### PR TITLE
test: home ts

### PR DESCRIPTION
**Pull request by ErinHales**

### CODE REVIEW

File: typescript/optics-tests/test/home.test.ts [hunk 1]
```diff
@@ -1,9 +1,258 @@
...
+    const newRoot = await enqueueMessageAndGetRoot('message');
+    const { signature: fakeSignature } = await fakeUpdater.signUpdate(
+      currentRoot,
+      newRoot,
+    );
+    await expect(
+      home.proxy.update(currentRoot, newRoot, fakeSignature),
+    ).to.be.revertedWith('bad sig');
+  });
+  it('Fails on valid double update proof', async () => {
```
Comment (prestwich) [inline typescript/optics-tests/test/home.test.ts:229]: leave a blank line between the tests

**Review by prestwich — CHANGES_REQUESTED** [review]

File: typescript/optics-tests/test/home.test.ts [hunk 2]
```diff
@@ -1,9 +1,268 @@
...
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
 const { expect } = require('chai');
-const { optics } = require('hardhat');
+
+import * as chain from '../../optics-deploy/src/chain';
+import * as deploys from '../../optics-deploy/src/deployOptics';
+import { getTestDeploy } from './testChain';
+import { OpticsState, Updater } from '../lib';
+
+const UpdaterManager = require('../../../solidity/optics-core/artifacts/contracts/UpdaterManager.sol/UpdaterManager.json');
```
Comment (prestwich) [inline typescript/optics-tests/test/home.test.ts:11]: can we change these to not be requires so we're not mixing syntax. you can import json in TS by adding ` "resolveJsonModule": true,` to the compiler options in the `.tsconfig`

**Review by prestwich — APPROVED** [review]

**Review by prestwich — COMMENTED** [review]
looking good, still need to change requires to imports

**Comment by ErinHales** [comment]
> looking good, still need to change requires to imports

I thought I got all of them...  Looks like I missed hardhat.config.ts, but I don't think there are any others?

**Pull request merged**